src/DeviceSelector.tsx

```diff
@@ -4,107 +4,114 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
  */
 
-import { connect } from 'react-redux';
-import path from 'path';
+import React from 'react';
+import { useDispatch } from 'react-redux';
 import {
     Device,
     DeviceSelector,
+    DeviceSetupConfig,
     getAppDir,
+    jprogDeviceSetup,
     logger,
-} from 'pc-nrfconnect-shared';
+} from '@nordicsemiconductor/pc-nrfconnect-shared';
+import { DeviceTraits } from '@nordicsemiconductor/pc-nrfconnect-shared/typings/generated/nrfutil';
```

Comment:
Is this type not imported from shared properly?
You're importing it from the generated typings.